qiita_db/support_files/qiita-db.dbs

```diff
@@ -1854,4 +1854,4 @@ ALTER TABLE oauth_software ADD CONSTRAINT fk_oauth_software FOREIGN KEY ( client
 ]]></string>
 		</script>
```

Comment:
This is no right, correct? There should have been changes in this file, cause we added a new column. Could you double check?

Reply:
Just added a line. Just to clarify, this is an internal format right? I don't see a schema reference at the top.